packages/pwa/app/components/order-summary/index.jsx

```diff
@@ -58,11 +58,11 @@ const CartItems = ({basket}) => {
                             >
                                 <Flex width="full" alignItems="flex-start">
                                     <CartItemVariantImage width="80px" mr={2} />
-                                    <Stack spacing={1} marginTop="-3px">
+                                    <Box flex="1" spacing={1} marginTop="-3px">
                                         <CartItemVariantName />
                                         <CartItemVariantAttributes includeQuantity />
                                         <CartItemVariantPrice baseDirection="row" />
-                                    </Stack>
+                                    </Box>
```

Comment:
Hey, thanks for the fix. Looking at it, it wasn't clear to me why using a box fixed the issue. So I played around with it a little and found that simply setting the child stack "width" property to `full` also had the same desired outcome. What do you think?

```
<Stack width="full" spacing={1} marginTop="-3px">
    <CartItemVariantName />
    <CartItemVariantAttributes includeQuantity />
    <CartItemVariantPrice baseDirection="row" />
</Stack>
```

Reply:
I agree with you. Keeping it as Stack also makes it more consistent to the rest of the code and makes much more sense. I guess, I was in a different state of mind when I attempted the fix. Apologies for that

Reply:
No problem. 👍